DatacodeStandard/Sniffs/Functions/ShortFunctionSpacingSniff.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DatacodeStandard\Sniffs\Functions;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Checks there is a space after a short function (fn)
+ *
+ * Same as the Squiz standard, but skips validation for @inheritDoc comments
```

Comment:
This comment doesn't really make sense for this sniff - looks like it was copied from the doc comments one.

Reply:
nooob! XD